File: LUA/tpl-0002.lua
-- TEXTPATGEN-QUICKSCRIPT-LUA
-- Please edit this file to suit your preferences.
-- The text is written to a timestamped file.

filename = os.date("00_%Y%m%d_%H%M%S.txt")
file = io.open(filename, "w")
introstr = os.date("TEXTPATGEN-QUICKSCRIPT-LUA: %Y-%m-%dT%H:%M:%S%z\n")
file:write(introstr)
n3 = 0
for n1 = 0, 9 do
  for n2 = 0, 8 do
    str = string.format("D-%05d ", n3)
    file:write(str)
    n3 = n3 + 1
    end 
  str = string.format("D-%05d\n", n3)
  file:write(str)
  n3 = n3 + 1
end
file:close()

File: RUST/tpl-0002.rs
// -- TEXTPATGEN-QUICKSCRIPT-RUST
// -- Please edit this file to suit your preferences.

use std::fs::File;
use std::io::Write;

fn main() -> std::io::Result<()> {
  let mut file = File::create("00_.txt")?;  // Please set the filename
  write!(file, "TEXTPATGEN-QUICKSCRIPT-RUST\n")?;
  let mut n3 = 0;
  for _n1 in 0..10 {
    for _n2 in 0..9 {
      write!(file, "D-{:05} ", n3)?;
      n3 = n3 + 1;
    }
    write!(file, "D-{:05}\n", n3)?;
    n3 = n3 + 1;
  }
  file.flush()?;
  Ok(())
}

